tests/test_components.py

```diff
@@ -84,3 +86,23 @@ def test_real_estate_area():
     RealEstateArea(
         value=123.45, unitCode="SQMT", raw="123.45 square meters", areaType="LOT"
     )
+
+
+def test_headers():
+    Header(name="Connection", value="keep-alive")
+    Header(name="Content-Type", value="text/html; charset=utf-8")
+
+
+def test_navigation():
+    Navigation(url="https://example.com")
+    headers = [
+        Header(name="Content-Type", value="application/x-www-form-urlencoded"),
+        Header(name="Host", value="foo.example"),
+    ]
+    Navigation(
+        name="Post Test",
+        url="https://example.com/test",
+        method="POST",
+        body="field1=value1&field2=value2",
```

Comment:
Isn't the body base64-encoded in the schema?

Reply:
We don't need to address it in this PR, but it'd be nice to have a more user-friendly way to set the body in Navigation items. E.g. auto-convert bytes to base64. But it's not 100% clear if we should use attrs convertors or web-poet processors for this, so to speed up the merge we can address it separately.

Reply:
Yes, it should be base64 encoded.

Reply:
Fixed